### ISSUE TITLE
Reference to the Codidactyl in non-Codidact instances

### ISSUE THREAD

**Issue by trichoplax** [codidact/qpixel#1396]
**Describe the bug**
When viewing the user page for a deleted user, the following message is displayed, which may be confusing for a user of an instance other than Codidact:

> User accounts make an excellent snack for the Codidactyl.

**To Reproduce**
1. Create a user account in a development environment.
2. Delete the user, making a note of their user id.
3. View your own user page.
4. Change the URL to use the deleted user's user id instead of your own.
5. See a deleted user message, including the message mentioning the Codidactyl.

**Expected behavior**
Hardcoded messages should be suitable for all instances, not just Codidact. Anything that only applies to Codidact should be conditional, so that it only appears when viewed from the Codidact instance.

Personally, I would also question whether we should have in-jokes at all in the user interface as the Codidact communities grow, as new users will have no context.

**Comment by cellio** [comment]
Agreed this should be configuration or generic.  It's fine to have a "mascot" that shows up in passing, and maybe we should have a "what is the Codidactyl?" help page that we can link to from such mentions, but if it gets in the way of comprehension, we should adjust.  I love the Codidactyl, but we want it to be friendly, not befuddling.
